### DISCUSSION TITLE
Is there a way to link mv-app and a mv-bar that is outside the app?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#888] [Q&A]
Hi

When I place mv-bar outise the mv-app, CSS styles  of course applies

but is there a way to indicate that this specific mv-bar is related to a specific mv-app?

[codepen](https://codepen.io/galinhalx/pen/bGvqjZg)

**Comment by LeaVerou** [comment]
No, this is currently not possible, though there are some CSS tricks that could help *position* the mv-bar to appear like it's outside.
If I may ask, what is your use case? Why not just place mv-app higher up?

**Reply by GalinhaLX** [reply]
Hi @LeaVerou 

I have a layout with multiple mv-apps

something like this:

Fixednavbar
............................
aside     /     main
............................
fixed bottom bar

I want to put "main" mv-bar inside "Fixednavbar"
but "fixednavbar" is already a mv-app
(in fact all 4 'slots' are mv-apps)

Layout is interactive to toggle "aside", "fixedtopbar" & "fixed bottom bar", and I can not move them in html :(

**Reply by GalinhaLX** [reply]
Somehow solved my goal:

Idea is to put custom buttons on 'Fixednavbar' with mv-action's that target other mv-app

**Reply by GalinhaLX** [reply]
Except the only ones I could reproduce were:

- clear
- edit

Is there a way to reproduce with mv-action or with mavo/javascript function call the actions behind:

- save
- import
- export
- login & logout

to, for example, save data of mv-app n°2 from a button inside mv-app n°1?
